tests/broker-engine/external-commands.robot

```diff
@@ -17,7 +17,7 @@ Library	../resources/Common.py
 
 *** Test Cases ***
 BEEXTCMD1
-	[Documentation]	external command CHANGE_NORMAL_SVC_CHECK_INTERVAL on bbdo3.0
+	[Documentation]	external command CHANGE_NORMAL_SVC_CHECK_INTERVAL on bbdo3.0, using bbdo3.0 the script configures the engine and broker, starts them, executes external commands to change the normal service check interval, verifies the interval values in the database, and stops the engine and broker. The process is repeated for different use_grpc values.
```

Comment:
We don't need to specify we stop engine/broker. We know that.
You can remove what we wrote before and make something better.
If I remember well, the test is made twice, the first time with the legacy external commands mechanism and the second time with grpc.